python/tox.ini

```diff
@@ -39,7 +39,7 @@ omit =
   *.jinja2
 
 [report]
-fail_under = 90.00
+fail_under = 89.00
```

Comment:
This might get to be put back to 90 if that test in `test_rename_provider` was removed by mistake!

Reply:
Maybe.